x/stakeibc/keeper/hooks.go

```diff
@@ -208,8 +208,8 @@ func (k Keeper) ReinvestRewards(ctx sdk.Context) {
 
 	for _, hostZone := range k.GetAllHostZone(ctx) {
 		// only process host zones once withdrawal accounts are registered
-		withdrawalIca := hostZone.WithdrawalAccount
-		if withdrawalIca == nil {
+		withdrawalAccount := hostZone.WithdrawalAccount
+		if withdrawalAccount == nil || withdrawalAccount.Address == "" {
```

Comment:
What's the case where address is an empty string?

Reply:
Shouldn't be possible in practice! But an empty/un-initialized `WithdrawalAccount` struct has an empty string for the `Address` field, so I think we should include it in the case where we check for a nil Account to be safe

```go
w := icatypes.WithdrawalAccount{} 
// w.Address is empty string 
```